app/views/posts/index.html.erb

```diff
@@ -1,13 +1,31 @@
-<h1>
-  All posts
-</h1>
+<table class="table">
+  <tbody>
+    <tr>
+      <td>
+        <%= image_tag @user.photo, alt: 'User Photo', class: 'user-photo' %>
+      </td>
+      <td>
+        <h1><%= @user.name %></h1>
+        <p><strong>Number of posts:</strong> <%= @user.posts_counter %></p>
```

Comment:
- [ ] Kindly ensure that your design matches the given design in terms of elements and layouts to meet the requirements.

- Kindly ensure your Number of posts is present and aligned to the right as is in the given design
- Your design should follow the following wireframe

Given
![image](https://github.com/Eccb7/Pi-Blogs/assets/81932478/2a5e97b7-b1b2-467b-90d3-a0bab2049c67)

Your design
![image](https://github.com/Eccb7/Pi-Blogs/assets/81932478/cafbb2c4-9264-40ce-bb3c-d584f58e71dc)

- Please use the recent comments method in your Post model to display the 5 most recent comments by a user